jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletApiRequest.java

```diff
@@ -1314,7 +1314,10 @@ else if (MimeTypes.Type.MULTIPART_FORM_DATA.is(baseType) &&
                             String msg = "Unable to extract content parameters";
                             if (LOG.isDebugEnabled())
                                 LOG.debug(msg, e);
-                            throw new RuntimeIOException(msg, e);
+
+                            if (cause instanceof IOException ioe)
+                                throw new UncheckedIOException(msg, ioe);
+                            throw new RuntimeException(msg, e);
```

Comment:
Use the `ExceptionUtil` method here?

Reply:
We can't really if we want to always wrap with the `msg`